ustriage/ustriage.py

```diff
@@ -317,7 +317,9 @@ def print_bugs(tasks, open_in_browser=0, shortlinks=True, blacklist=None,
             reverse=oder_by_date
         )
 
-    logging.info('Found %s bugs', len(sorted_filtered_tasks))
+    logging.info('Found %s bugs\n', len(sorted_filtered_tasks))
+
+    logging.info(Task.get_header(extended=extended))
```

Comment:
Would it make more sense to use the same mechanism for outputting the header and the data?

`compose_pretty` is being dumped with `print` and the header through logging, if, for instance, one changes the log file, we will have a header without data (and data without a header).

Reply:
yeah, makes sense.
I unified them all (was a bigger change) in an extra commit.